ssn/integrated/ssn.ttl

```diff
@@ -37,7 +37,24 @@ vann:preferredNamespaceUri a owl:AnnotationProperty .
 ssn: a owl:Ontology ;
   owl:imports sosa: .
 
+## Features of interest and Property
 
+#sosa:FeatureOfInterest
+#  rdfs:subClassOf [ owl:onProperty ssn:hasProperty ; owl:someValuesFrom ssn:Property ] ; # TODO: need resolution, is it ssn:hasProperty, or sosa:hasProperty ?
+#  rdfs:subClassOf [ owl:onProperty ssn:hasProperty ; owl:allValuesFrom ssn:Property ]  . # TODO: need resolution, is it ssn:hasProperty, or sosa:hasProperty ?
+
```

Comment:
need resolution, is it ssn:hasProperty, or sosa:hasProperty ?
This is currently discussed in a separate thred, will just comment these lines for now.
See https://lists.w3.org/Archives/Public/public-sdw-wg/2017Feb/0478.html